views/pages/objects/grid.blade.php

```diff
@@ -0,0 +1,37 @@
+@extends('layout.containers.doc')
+
+@section('doc-content')
+    @markdown
+        #Grid
+        Grid is a object that allows you to create a grid of elements. Please note that we support, .grid and .o-grid syntax. We are moving away from .grid class to the better scoped .o-grid. 
```

Comment:
Started as a grammatical fix but I ended up having further suggestions for clarifying this paragraph, do with it as you wish. 
```suggestion
        Grid is an object that allows you to create a grid of elements. Please note that we support both the .grid and .o-grid syntax. However the .grid class should be considered deprecated and .o-grid  as it's better scoped and is therefor favored.
```